lib/initialize.js

```diff
@@ -1,26 +1,14 @@
-const inquirer = require('inquirer');
 const questions = require('./utils/observable-questions');
 const validateAddons = require('./utils/validate-addons');
-const parse = require('./parser/index');
+const inq = require('./inquirer-prompt');
+const initialConfig = require('./utils/initial-config');
+const Rx = require('rx');
 
 module.exports = function initializeInquirer(pkg) {
 	if(pkg.length == 0) {
-		//eslint-disable-next-line
-		let entry,output = {};
-		inquirer.prompt(questions).ui.process.subscribe(
-			(ans) => {
-				if(ans.name === 'entry') entry = ans.answer;
-				else output = ans.answer;
-			},
-			(err) => {
-				console.log('Error: ', err);
-			},
-			function inquirerParser() {
-				parse(null, {entry,output});
-			}
-		);
-	} else {
-		validateAddons(pkg);
-		// hook parser with promise?
+		return inq(Rx.Observable.from(questions), initialConfig);
```

Comment:
Let's don't say bytes and be more specific on variable names. `prompt` would be a better fit IMO. Also, it should be consistent across all files.